pkg/network/node/iptables.go

```diff
@@ -27,6 +28,23 @@ type NodeIPTables struct {
 	egressIPs map[string]string
 }
 
+var iptablesBackoff wait.Backoff = wait.Backoff{
+	Duration: 500 * time.Millisecond,
+	Factor:   1.25,
+	Steps:    10,
+}
+
+// execIPTablesWithRetry allows a simple way to retry iptables commands if they fail the first time
+func execIPTablesWithRetry(f func() error) error {
+	return wait.ExponentialBackoff(iptablesBackoff, func() (bool, error) {
+		if err := f(); err != nil {
+			klog.V(5).Infof("Call to iptables failed: %v", err)
+			return false, nil
```

Comment:
This will retry on _any_ iptables failure, but we really only want to retry on "temporary"/"transient" failures; make a copy of `isResourceError` from `vendor/k8s.io/kubernetes/pkg/util/iptables/iptables.go`

Reply:
done